### PR TITLE
feat(bors) Set up Bors

### PR DESCRIPTION
**Pull request by Hywan**
I'm trying to setup Bors. I think we may update some settings on the Github repo. See https://bors.tech/documentation/getting-started/ for the documentation.

### CODE REVIEW

**Comment by Hywan** [comment]
bors try

**Comment by lachlansneff** [comment]
bors ping

**Comment by lachlansneff** [comment]
bors try

**Comment by xmclark** [comment]
bors hello

File: .appveyor.yml [hunk 1]
```diff
@@ -2,8 +2,10 @@ version: "{build} ~ {branch}"
 
 os: Visual Studio 2017
 
-# Do not build feature branch with open Pull Requests
-skip_branch_with_pr: true
+branches:
+  only:
+    - staging
+    - trying
```
Comment (syrusakbary) [inline .appveyor.yml]: @Hywan could be possible to know why we only want to test only in `staging` or `trying` branches?

**Comment by lachlansneff** [comment]
bors +r

**Comment by lachlansneff** [comment]
yes, sorry, I'm new to this
bors r+

File: .appveyor.yml (hunk 1 above)
Comment (lachlansneff) [reply .appveyor.yml]: That's what you're supposed to do: https://bors.tech/documentation/getting-started/

**Comment by lachlansneff** [comment]
bors try

**Comment by lachlansneff** [comment]
bors try

File: .appveyor.yml (hunk 1 above)
Comment (Hywan) [reply .appveyor.yml]: Correct.

**Pull request merged**